tests/xfstests/enable_kdump.pm

```diff
@@ -26,8 +26,11 @@ sub run {
     select_console('root-console');
 
     # Also panic when softlockup
-    assert_script_run('echo "kernel.softlockup_panic = 1" >> /etc/sysctl.conf');
-    assert_script_run('sysctl -p');
+    # workaround bsc#1104778, skip s390x in 12SP4
+    unless (check_var('ARCH', 's390x') && check_var('VERSION', '12-SP4')) {
```

Comment:
Let's use is_sle call to verify version, so it will be is_sle('=12-SP4') here.
In general I would also avoid complex conditions with `unless`
Also, what's about adding soft-failure in case of s390x on SP4? So we won't forget about that bug once it get's fixed.